cmake/defaults/Packages.cmake

```diff
@@ -251,7 +258,22 @@ if (PXR_BUILD_IMAGING)
     endif()
     # --Opensubdiv
     set(OPENSUBDIV_USE_GPU ${PXR_BUILD_GPU_SUPPORT})
-    find_package(OpenSubdiv 3 REQUIRED)
+    find_package(OpenSubdiv 3 CONFIG)
+    if(OpenSubdiv_DIR)
+        set(OPENSUBDIV_LIBRARIES OpenSubdiv::osdCPU OpenSubdiv::osdGPU)
+        set(OPENSUBDIV_OSDCPU_LIBRARY OpenSubdiv::osdCPU)
+        message(STATUS "OpenSubdiv found as config, OpenSubdiv_DIR is ${OpenSubdiv_DIR}")
+        foreach(t OpenSubdiv::osdCPU OpenSubdiv::osdGPU OpenSubdiv::osdCPU_static OpenSubdiv::osdGPU_static OpenSubdiv::OpenSubdiv)
+            if(TARGET "${t}")
+                message(STATUS "${t} is TARGET")
+            else()
+                message(STATUS "${t} is not TARGET")
+            endif()
+        endforeach()
```

Comment:
These messages feel more like debugging messages than things that need to be displayed every build - do you think we could lower the message code from `STATUS` to  `VERBOSE` or `DEBUG`?

Reply:
I'm sorry, they are debugging messages and I removed them in my final commit. I set up a workflow on my fork and I was experimenting with solutions to fix the build error here.

Sorry for the noise, I left the PR in Draft state to indicate it's in progress.

Reply:
removed